update_appdata.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# replace AppID in appdata file using '|' as sed delimiter
+sed -i 's|<id>org.mavlink.qgroundcontrol</id>|<id>org.mavlink.QGroundControl</id>|g' $1
+# update licence
+sed -i 's|<metadata_license>GPL-3.0 AND Apache-2.0</metadata_license>|<metadata_license>CC0-1.0</metadata_license>|g' $1
```

Comment:
you can NOT do that. File an issue with upstream.

Reply:
I know. I proposed that change here: https://github.com/mavlink/qgroundcontrol/pull/10615

I kept this PR as a draft because of this licence issue. But I still wanted to see the CI pass.

Reply:
as I said above: this should be a patch, not script with obscure sed commands.